### DISCUSSION TITLE
Feature request: allow specify monospace font family

### DISCUSSION THREAD

**Discussion by reorx** [notaapp/talk#305] [Feature Requests] [answered]
Problem: CJK characters in markdown table could not be aligned.

![image](https://user-images.githubusercontent.com/405972/189387774-8da5da2c-3c75-411f-9321-feac218736b1.png)

This is due to most of the monospace fonts do not contain CJK characters, leaving CJK characters rendered by the default font, the width is not equal to 1 or 2 monospace characters.

[Sarasa Gothic](https://github.com/be5invis/Sarasa-Gothic) and Noto Sans CJK are monospace fonts with CJK characters, but they are not very suit for writing. The best solution is to set font family for monospace separately.

**Comment by astoilkov** [comment, marked as answer]
I agree. We should add such an option. Thanks for explaining your problem so well.
